### DISCUSSION TITLE
A few ideas

### DISCUSSION THREAD

**Discussion by introvertedcreature** [mifi/lossless-cut#1269] [Ideas]
-Add an option to scan the whole video (keyframes & audio waveform) upon opening
-Hide the audio waveform if there's no audio track
-Some options reset between restarts, avoid that if possible

Sorry if some/all of these have been suggested already

**Comment by mifi** [comment]
> -Add an option to scan the whole video (keyframes & audio waveform) upon opening

see #260

> -Hide the audio waveform if there's no audio track

I have improved this now, thanks

> -Some options reset between restarts, avoid that if possible

#371
